modules/ROOT/nav.adoc

```diff
@@ -33,7 +33,7 @@
 *** xref:kernel-args.adoc[Modifying Kernel Arguments]
 *** xref:alternatives.adoc[Setting alternatives]
 ** OS updates
-*** xref:update-streams.adoc[Update Streams]
+*** xref:available-update-streams.adoc[Update Streams]
```

Comment:
In general, we try to avoid renaming pages because it breaks links (obviously it's fine if the page is substantially changed or deleted). Probably not worth it in this case?